sap-schemas/v3.0/README.md

```diff
@@ -84,6 +84,13 @@ Constraints:
     | ---------- | -------------------- | --------------------------------------------------- |
     | name       | `String`             | **REQUIRED**. Information title                     |
     | values     | `String` or `Object` | **REQUIRED**. Object type MAY contain any structure |
```

Comment:
@pavelkornev : Do we still allow values to be a simple string as well? 

This also states "Object type MAY contain any structure" and below we state how it "may" look. 

I think we need to state this stronger, as there's only certain structures that are understood by the Business Hub. 

Maybe like so:

```suggestion
    | values     | `String` or `Object` | **REQUIRED**. Object MUST follow the structure like described below |
```